commands/actor.go

```diff
@@ -47,7 +49,8 @@ var actorCmd = &cmds.Command{
 		Tagline: "Interact with actors. Actors are built-in smart contracts.",
 	},
 	Subcommands: map[string]*cmds.Command{
-		"ls": actorLsCmd,
+		"ls":          actorLsCmd,
+		"power-table": actorPowerCmd,
```

Comment:
Only miner actors can have power, and it would be more findable if this command were with the existing `miner power` command. Please move this to the `miner` command. Move the tests appropriately.